sgqlc/codegen/schema.py

```diff
@@ -92,11 +92,7 @@ def to_docstring(wrapped_text, level=1):
     if not wrapped_text:
         return ''
     prefix = '    ' * level
-    if len(wrapped_text) == 1:
-        suffix = ''
-    else:
-        suffix = '\n' + prefix
-
+    suffix = '' if len(wrapped_text) == 1 else '\n' + prefix
```

Comment:
this one makes it shorter for sure, but I really dislike these ternary in Python as it reminds me of Perl.